metrics/web/lib/base/presentation/graphs/circle_percentage.dart

```diff
@@ -2,12 +2,12 @@ import 'dart:math';
 
 import 'package:flutter/material.dart';
 
-/// The widget that represents the metric percent in a circular graph.
+/// The widget that represents the metrics percent in a circular graph.
```

Comment:
```suggestion
/// The widget that represents the percent in a circular graph.
```
as this is a Base widget - it shouldn't contain any references to Metrics - so I feel we need to remove it altogether.